### PR TITLE
Add client side password validation to user management in the Console

### PR DESCRIPTION
**Pull request by bafonins**
#### Summary

References https://github.com/TheThingsNetwork/lorawan-stack/issues/1256

<details>
<summary>Screenshots</summary>
Length:

Screenshot 2020-11-15 at 12 44 40
Screenshot 2020-11-15 at 12 40 48

Special:

Screenshot 2020-11-15 at 12 41 00
Screenshot 2020-11-15 at 12 45 59

Digits:

Screenshot 2020-11-15 at 12 41 13
Screenshot 2020-11-15 at 12 47 02

Uppercase:

Screenshot 2020-11-15 at 12 41 25
Screenshot 2020-11-15 at 12 46 57

</details>

#### Changes

- Add `unicode-properties` dependency
- Add password utils
- Add password utils tests
- Add length, special, digits and uppercase validation to the user management

#### Testing

A lot of manual testing + unit tests

#### Notes for Reviewers

1. Didnt touch the oauth views because of the account app work in progress. With this PR in place adding client side password validation to account app is just copy pase. UPD: it also takes more time to add to the `create-account` view in the oauth app and will definitely introduce merge conflicts with the account branch changes. So I think it is better to add it after the account app branch gets merged.
2. `codePointAt` is not supported in IE11, but even without this console/oauth do not work in IE11 anyway 

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/lib/password/password_test.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  })
+
+  describe('hasDigit', () => {
+    const tests = [
+      { result: true },
+      { password: 'no-digits', digits: 0, result: true },
+      { password: '1', digits: 0, result: true },
+      { password: '1', digits: 1, result: true },
+      { password: '٩', digits: 1, result: true },
+      { password: '1️⃣', digits: 1, result: true },
```
Comment (kschiffer) [inline pkg/webui/lib/password/password_test.js:68]: Is this really how the rule is interpreted also in the backend? I feel like such special character should not be interpreted as number.

**Review by kschiffer — CHANGES_REQUESTED** [review]
Code LGTM. Please add the following warning to the password field whenever non-ASCI-256 characters are added:
`Adding non-Latin characters might cause issues when having to enter the password on devices without such input capabilities`

File: pkg/webui/lib/password/password_test.js (hunk 1 above)
Comment (bafonins) [reply pkg/webui/lib/password/password_test.js:68]: https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/pkg/identityserver/user_registry.go#L91
https://golang.org/pkg/unicode/#pkg-variables, see `Nd`
https://apps.timwhitlock.info/unicode/inspect?s=1%EF%B8%8F%E2%83%A3 - see the `Cat` column

Comment (bafonins) [reply pkg/webui/lib/password/password_test.js:68]: Also see discussion in https://github.com/TheThingsNetwork/lorawan-stack/pull/1239

**Comment by bafonins** [comment]
> Please add the following warning to the password field whenever non-ASCI-256 characters are added

If you think that this is needed, please create a separate issue for this particular feature. It is not as simple as just checking whether character code of a certain character is between 0-255. As an example, `'1️⃣'` has char code of 49 similar to regular `1`.

**Review by kschiffer — APPROVED** [review]

**Pull request merged**